src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -482,12 +483,14 @@ function WebRTCAdaptor(initialValues)
 	}
 
 	this.join = function(streamId) {
+		thiz.streamId = streamId;
```

Comment:
Do we need to define a new inline object variable?
If yes, we need to make available in the list above. Otherwise it will be very hard to maintain the code.
If no, just remove it.